### PR TITLE
Show git path in gem env

### PR DESCRIPTION
**Pull request by bronzdoc**
# Description:
closes https://github.com/rubygems/rubygems/issues/2021

Adds Git path to `gem env`
```
RubyGems Environment:
  ...
  - GIT EXECUTABLE: /usr/bin/git
  ...
```

still need to add a test for this...

I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: lib/rubygems.rb [hunk 1]
```diff
@@ -846,6 +846,21 @@ def self.prefix
     end
   end
 
+  ##
+  # Git binary path
+
+  def self.git_path
```
Comment (segiddins) [inline lib/rubygems.rb]: I think we might want to make this a private method in `environment_command.rb`, since that's the only place it is being used

Comment (bronzdoc) [reply lib/rubygems.rb]: alright :+1:

**Comment by bronzdoc** [comment]
I updated the PR @segiddins let me know what you think, so i can squash the commits, thanks

File: lib/rubygems.rb (hunk 1 above)
Comment (segiddins) [reply lib/rubygems.rb]: could call `send` in the tests ?

**Comment by bronzdoc** [comment]
@bundlerbot r+

**Pull request merged**